### PR TITLE
parser, executor: implement plan replayer capture statement

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb/issues/38779

Problem Summary:

### What is changed and how it works?

implement plan replayer capture statement

### Check List

Tests 

- [x] Unit test

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: executor/plan_replayer.go [hunk 1]
```diff
@@ -89,6 +100,20 @@ func (e *PlanReplayerExec) Next(ctx context.Context, req *chunk.Chunk) error {
 	return nil
 }
 
+func (e *PlanReplayerExec) registerCaptureTask(ctx context.Context) error {
+	ctx1 := kv.WithInternalSourceType(ctx, kv.InternalTxnStats)
+	exec := e.ctx.(sqlexec.SQLExecutor)
+	_, err := exec.ExecuteInternal(ctx1, fmt.Sprintf("insert into mysql.plan_replayer_task (sql_digest, plan_digest) values ('%s','%s')",
```
Comment (qw4990) [inline executor/plan_replayer.go:106]: How about checking whether it's duplicated first here and returning an error msg to notify the user if it is?

File: executor/executor_test.go [hunk 2]
```diff
@@ -174,6 +174,17 @@ func TestPlanReplayer(t *testing.T) {
 	require.Len(t, rows, 1)
 }
 
+func TestPlanReplayerCapture(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("plan replayer capture '123' '123';")
+	tk.MustQuery("select sql_digest, plan_digest from mysql.plan_replayer_task;").Check(testkit.Rows("123 123"))
+	_, err := tk.Exec("plan replayer capture '123' '123';")
```
Comment (hawkingrei) [inline executor/executor_test.go:183]: ```tk.MustGetErrMsg```

**Review by qw4990 — APPROVED** [review]

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**